packages/suite/src/components/firmware/FirmwareOffer.tsx

```diff
@@ -114,81 +100,64 @@ const FirmwareOffer = ({ device, customFirmware, targetFirmwareType }: Props) =>
                         <Label>
                             <Translation id="TR_ONBOARDING_CURRENT_VERSION" />
                         </Label>
-                        <VersionWrapper>
-                            <Version>
-                                {previousFirmwareType}
-                                {currentVersion}
-                            </Version>
-                        </VersionWrapper>
+                        <Version>{currentTypeAndVersion}</Version>
                     </FwVersion>
-                    <IconWrapper>
-                        <Icon icon="ARROW_RIGHT_LONG" size={16} />
-                    </IconWrapper>
+                    <Icon icon="ARROW_RIGHT_LONG" size={16} />
                 </>
             )}
             <FwVersion>
                 <Label>
                     <Translation id="TR_ONBOARDING_NEW_VERSION" />
                 </Label>
-                <VersionWrapper>
-                    {parsedChangelog ? (
-                        <Tooltip
-                            rich
-                            dashed
-                            content={
-                                <Changelog>
-                                    <ChangelogGroup key={parsedChangelog.versionString}>
-                                        <ChangelogHeading>
-                                            <Translation
-                                                id="TR_VERSION"
-                                                values={{ version: parsedChangelog.versionString }}
-                                            />
-                                            <TrezorLink
-                                                size="small"
-                                                variant="nostyle"
-                                                href={
-                                                    parsedChangelog.notes
-                                                        ? parsedChangelog.notes
-                                                        : changelogUrl
-                                                }
+                {parsedChangelog ? (
+                    <Tooltip
+                        rich
+                        dashed
+                        content={
```

Comment:
Perhaps the content could be taken out for a cleaner render?

Reply:
Yes, I turned it into a component in 05d94686197da26dd84461fc9a927226b60b5cc2.